Makefile

```diff
@@ -0,0 +1,33 @@
+BOARD ?= m3
+
+.PHONY: build-docker-image build-docker-image-test setup-ude-rules \
```

Comment:
just notice that there's a typo on `setup-ude-rules` (missing `v` in ude**v**): make autocompletion spotted this.

Reply:
I saw that, didn't know where the typo was :-)

Reply:
it's now fixed